hide/comp/SceneEditor.hx

```diff
@@ -1070,18 +1077,26 @@ class SceneEditor {
 		}
 
 		var map = new Map<PrefabElement,Bool>();
-		function selectRec(e : PrefabElement, b:Bool) {
+		function select(e : PrefabElement, b:Bool) {
 			if( map.exists(e) )
 				return;
 			map.set(e, true);
 			var ectx = context.shared.contexts.get(e);
 			setObjectSelected(e, ectx == null ? context : ectx, b);
+		}
```

Comment:
@ncannasse what do you think of making `setObjectSelected` return false if we want to stop the recursive selection?